### PR TITLE
Fix: DesiredRole field regex updated to allow spaces (#114)

### LINKED ISSUE #114: Bug: Desired Role field cannot accept '/'
# Objective
Change the desired role regex as it cannot accept '/'. I already tried different prompts. Refer to the images below. 

# Context
image

### PR DESCRIPTION
**Pull request by dominic2412**
close #114 
**Objective**  
This pull request addresses an issue where the **DesiredRole** field did not accept spaces, causing inconvenience when users attempted to add multiple-word skills.

**Changes Made**  
- Updated the regex pattern in the **DesiredRole** field 
- Tested the regex update to ensure it handles both single-word and multi-word inputs correctly.
- Desired role doesn't accept "/"

**Review Notes**  
- Please review the regex changes and verify that it covers various scenarios.
- Let me know if additional test cases are needed for edge cases.

### CODE REVIEW

File: src/main/java/seedu/address/model/person/DesiredRole.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class DesiredRole {
     /*
      * The first character of the desired role must not be a whitespace.
      */
-    public static final String VALIDATION_REGEX = "[A-Za-z][A-Za-z0-9 ]*";
+    public static final String VALIDATION_REGEX = "^[A-Za-z][A-Za-z0-9 /]*$";
```
Comment (Darren-Tung) [inline src/main/java/seedu/address/model/person/DesiredRole.java:17]: Good regex
